package/gluon-core/luasrc/lib/gluon/upgrade/820-dns-config

```diff
@@ -14,6 +14,20 @@ uci:set('dhcp', dnsmasq, 'localservice', false)
 uci:set('dhcp', dnsmasq, 'server', dns.servers)
 uci:set('dhcp', dnsmasq, 'cachesize', dns.cacheentries)
 
+if dns.servers then
+	uci:set('dhcp', dnsmasq, 'server', dns.servers)
```

Comment:
This line is duplicated, once inside and once outside of the *if*. The one outside the *if* should be kept.

Reply:
done.